grails-app/views/species/show.gsp

```diff
@@ -46,7 +46,7 @@
     </head>
 
     <body>
-        <div class="container-fluid page-taxon">
```

Comment:
We shouldn't just ditch the container classes from everwhere. Bootstrap demands them.
https://v4-alpha.getbootstrap.com/layout/overview/#containers

I'm guessing the padding is cancelled by ```row``` classes, which are not used in our page headings. There's no point in wrap the page headings in rows, for they do not use Bootsrap's layout. Unless, of course, we decide that they should.

Would it be possible to strart the main container after the page heading section?